airflow/providers/google/cloud/operators/pubsub.py

```diff
@@ -674,3 +675,130 @@ def execute(self, context):
         self.log.info("Publishing to topic %s", self.topic)
         hook.publish(project_id=self.project_id, topic=self.topic, messages=self.messages)
         self.log.info("Published to topic %s", self.topic)
+
+
+class PubSubPullOperator(BaseOperator):
+    """Pulls messages from a PubSub subscription and passes them through XCom.
+
+    This Operator always calls PubSub API with return_immediately=True.
+        This means that the graph execution will continue regardless of
+        whether there are any messages awaiting in the topic.
+        If you need to wait for messages, please use
+        :class:`airflow.providers.google.cloud.sensors.PubSubPullSensor`
+        instead.
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:PubSubPullSensor`
+
+    .. seealso::
+        :class:`airflow.providers.google.cloud.sensors.PubSubPullSensor`
```

Comment:
Can you add short sentences before link?

Reply:
Added